### DISCUSSION TITLE
New release ETA of glpi-agent

### DISCUSSION THREAD

**Discussion by kintaro1981** [glpi-project/glpi-agent#395] [Q&A] [answered]
Hello, I see that the develop branch have many fixes and the latest release is from July 1st 2022.

Is it possible to know an eta about the release of GLPI Agent (1.5?)?
I'm asking it because I'm near to migrate to GLPI Agent from Fusion Inventory agent and in case a new release is near I will wait a little bit longer.

Thank you

**Comment by g-bougard** [comment, marked as answer]
Hi @kintaro1981 

today, the answer is the same than ##373.

We are not near from a release as we still wait a new EV Code Signing key and it takes time. So I advise you to start your migration.
